dnaplotlib/sbol/sbolplotlib.py

```diff
@@ -15,54 +15,44 @@
 #    All rights reserved.
 #    OSI Open Software License 3.0 (OSL-3.0) license.
 
-import matplotlib
-import matplotlib.pyplot as plt
-
-from matplotlib.patches import Polygon, Ellipse, Wedge, Circle, PathPatch, Rectangle
-from matplotlib.path import Path
-from matplotlib.lines import Line2D
-from matplotlib.patheffects import Stroke
-import matplotlib.patches as patches
-import math
-
 import dnaplotlib as dpl
-import sbol
 
-__author__  = 'Bryan Bartley <bartleyba@sbolstandard.org>\n\
-               Thomas E. Gorochowski <tom@chofski.co.uk>'
-__license__ = 'MIT'
-__version__ = '1.0'
+__author__ = "Bryan Bartley <bartleyba@sbolstandard.org>\n\
+               Thomas E. Gorochowski <tom@chofski.co.uk>"
+__license__ = "MIT"
+__version__ = "1.0"
+
 
 class SBOLRenderer(dpl.DNARenderer):
 
     def SO_terms(self):
-        """ Return dictionary of all standard built-in SBOL part renderers referenced by Sequence Ontology term
-        """
+        """Return dictionary of all standard built-in SBOL part renderers referenced by Sequence Ontology term"""
         return {
-        'SO_0000167': 'Promoter',
-        'SO_0000316': 'CDS',
-        'SO_0000141': 'Terminator',
-        'SO_0000552': 'RBS',
-        'SO_0001953': 'Scar',
-        # No SO Term : 'Spacer',
-        # No SO Term : 'EmptySpace',
-        'SO_000037': 'Ribozyme',
-        'SO_0001977': 'Ribonuclease',
-        'SO_0001955': 'ProteinStability',
-        'SO_0001956': 'Protease',
-        'SO_0000057': 'Operator',
-        # SO term insulator does not have same semantics : 'Insulator',
-        'SO_0000296': 'Origin',
-        'SO_0001932': '5Overhang',
-        'SO_0001933': '3Overhang',
-        'SO_0001687': 'RestrictionSite',
-        'SO_0000299': 'RecombinaseSite',
-        'SO_0001691': 'BluntRestrictionSite',
-        'SO_0005850': 'PrimerBindingSite',
-        'SO_0001694': '5StickyRestrictionSite',
-        'SO_0001690': '3StickyRestrictionSite',
-        'SO_0000001': 'UserDefined',
-        'SO_0001978': 'Signature',
+            "SO:0000167": "Promoter",
+            "SO:0000316": "CDS",
+            "SO:0000141": "Terminator",
+            "SO:0000552": "RBS",
+            "SO:0000139": "RBS",
```

Comment:
I added this symbol http://www.sequenceontology.org/browser/current_release/term/SO:0000139
used the same label as it appears to be the parent of SO:0000552 in the hierarchy.